src/mjml/mjml.service.js

```diff
@@ -46,7 +46,11 @@ export default class MjmlService {
    * @todo show validation erros in the UI
    * @returns string
    */
-  static mjmlToHtml(mjml) {
+  static mjmlToHtml(mjml, endpoint = null) {
+    if (endpoint !== null) {
+      return MjmlService.mjmlToHtmlViaEndpoint(mjml, endpoint);
+    }
+
     try {
       if (typeof mjml !== 'string' || !mjml.includes('<mjml>')) {
         throw new Error('No valid MJML provided');
```

Comment:
I would add the endpoint part after the check if a valid string was passed in.

Reply:
otherwise you dont benefit from the existing validity check

Reply:
done